blakserv/ccode.c

```diff
@@ -1939,6 +1939,38 @@ blak_int C_GetTime(int object_id,local_var_type *local_vars,
 	return ret_val.int_val;
 }
 
+int C_GetTickCount(int object_id,local_var_type *local_vars,

```

Comment:
change the return type to `blak_int`

`blak_int C_GetTickCount` instead of `int C_GetTickCount`